packages/wonder-blocks-core/components/view.js

```diff
@@ -40,9 +51,20 @@ const StyledDiv = addStyle<"div">("div", styles.default);
  * - An `aphrodite` StyleSheet style
  * - An array combining the above
  */
-export default class View extends React.Component<TextViewSharedProps> {
+export default class View extends React.Component<Props> {
     render() {
-        const {testId, ...restProps} = this.props;
-        return <StyledDiv data-test-id={testId} {...restProps} />;
+        const {testId, tag, ...restProps} = this.props;
+        switch (tag) {
+            case "article":
```

Comment:
Now that we have explicit cases we need to test each of them separately in view.test.js.

Reply:
Is it possible to avoid using a switch statement here?

```
const StyledTag = addStyle<string>(tag, styles.default);
return <StyledTag data-test-id={testId} {...restProps} />;
```

Not sure if this is one of flow's limitations about type checking. Mostly b/c the tests won't fail unless we use `// @flow`.
cc @kevinbarabash

Reply:
Unfortunately that won't work b/c `tag` is typed as `"article" | "section" | ...` which incompatible with `string`.  Also, I think that calling `addStyle` in `render()` is probably bad practice since it creates a new class each time which feel expensive.